### ISSUE TITLE
Sassc compatibility

### ISSUE THREAD

**Issue by normanrz** [n33/skel#142]
Are there plans to support the libsass-based `sassc` compiler?

Currently (libsass 3.2.1) there is an error:
```
Error: invalid selector after 25
        on line 221 of src/_skel.scss
>> 			.container.\31 25\25		{ width: 100%; max-width: $width * 1.25; min-width: $
   ------------------^
```

I am not familiar with this backslash syntax so I don't really know how to help. Can you point me to a resource to learn about this?

Thanks!

**Comment by n33** [comment]
The backslashes are used to escape unicode values, and it's what enables Skel to use classes that begin with numbers (`12u`, `50%`, etc).

That said, I wasn't aware of any compatibility issues with sassc, but I'll look into it.

**Comment by marcusmoore88** [comment]
Hello!

I am also encountering this problem. I'm using the following gulp dependencies:
```
"gulp": "^3.9.0",
"gulp-autoprefixer": "^2.3.1",
"gulp-concat": "^2.5.2",
"gulp-debug": "^2.0.1",
"gulp-sass": "^2.0.1"
```

I get an error:
```
gulp-sass] sass/libs/_skel.scss
  204:19  invalid selector after 25
```

 when we get to this line in skel v3.0.0-dev: `.container.\31 25\25		{ width: 100%; max-width: $width * 1.25; min-width: $width; }` 

Do you have any advice on this?

**Comment by vebam** [comment]
I confirm,
Gulp + skel = error: invalid selector

**Comment by marcusmoore88** [comment]
n33, can you tell us what you use to compile your assets? I think in the short-term users might be able to adopt that workflow but in the long run it might be best to get gulp/sass working?

**Comment by vebam** [comment]
it looks like that will correctly with  Ruby Sass

**Comment by n33** [comment]
Yep, I'm using Ruby Sass, although I'd like to get it working with sassc as well.

**Comment by garrettshearer** [comment]
I can also confirm that this does not work with:

Webpack + Babel + sass-loader

**Comment by shai126** [comment]
My workaround for now, in case it helps others, has been to swap gulp-sass
for gulp-ruby-sass. The syntax is a little different, but does the job
great.

I'd be happy to look at fixing this though - I assume libsass is the place
to start?

**Comment by marcusmoore88** [comment]
There was an issue I found (I think in the libsass issue tracker) that was talking about numeric selectors but I can't seem to find it now. I believe that libsass is the place to start as well.

**Comment by LoganLehman** [comment]
I am having the same problem using brackets-sass. It uses libsass.

**Comment by n33** [comment]
Nice! Looks like this will be fixed in the next release of libsass:

https://github.com/sass/libsass/issues/1377

Will keep this issue open until it comes out though.

**Comment by marcusmoore88** [comment]
That's awesome. I don't have any immediate projects using skel but I've been putting off playing with it because of this.

**Comment by JesseLeite** [comment]
I'm having problems too :(

**Comment by ethixdesign** [comment]
@JesseLeite I got around this by compiling with Koala: http://koala-app.com

;)

**Comment by JesseLeite** [comment]
I just used ruby sass, all good.  I'm building with Laravel Elixir, and ruby sass is a compiler option.  The problem was spending hours trying to debug Pixelarity template, thinking it was issue with the .scss.  Instead it's issue with libsass / gulp-sass.

**Comment by Jdeezy** [comment]
as long as you got beyond the hurdle  ;)

**Comment by mohamnag** [comment]
I dont have a compile time problem but when using Usemin of grunt, the compressed css is brocken just from that first unicode character. I ask myself if what you have done could not be achieved using normal ascii characters that are safe to use every where?

**Comment by n33** [comment]
Yep, I use weird class names. Fixed in libsass 3.3.

**Comment by trawick** [comment]
For those who find this ticket while trying to webpack-ize skel with sass-loader (which uses node-sass which bundles libsass):

Another escaping problem has [re]appeared in more recent libsass.  Some bad vs. good examples in the generated .css:

```
-.12u,.12u24 {
+.12u,.12u\24 {
...
-.12u28desktop29,.12u2428desktop29 {
+.12u\28desktop\29,.12u\24\28desktop\29 {
```
With ugly hacks to _skell.scss (just one example: replace "u" with "\75 ") I can get a lot of the issues cleared away, but not enough.

```
node-sass	4.8.3	(Wrapper)	[JavaScript]
libsass  	3.5.2	(Sass Compiler)	[C/C++]
```

Downgrading towards a version of node-sass with the libsass version referenced here eventually results in v8 crashes.

It would be fun to play with libsass and open a ticket with a good testcase; unclear if/when I'll get around to that.
